aten/src/ATen/native/quantized/cpu/tensor_operators.cpp

```diff
@@ -59,7 +59,14 @@ AT_FORALL_OPERATORS(DEFINE_COMPARATOR)
 #undef AT_FORALL_OPERATORS
 #undef DEFINE_COMPARATOR
 
-Tensor& quantized_resize_cpu_(Tensor& self, IntArrayRef size) {
+Tensor& quantized_resize_cpu_(
+    Tensor& self,
+    IntArrayRef size,
+    c10::optional<MemoryFormat> optional_memory_format) {
+  TORCH_CHECK(
+      !optional_memory_format.has_value(),
```

Comment:
are you sure we need this?  quantized tensors should just be able to resized in the same way.